index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <nav id="nav-flex-container">
+      <p>Welcome</p>
+      <img src="Images/Menu.svg" alt="Menu" />
+    </nav>
+    <header id="header-flex-container">
+      <h1>
+        Hey there. I’m Kerolous <br /><span>I’m a software developer</span>
+      </h1>
+      <p>
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hestiate to contact me.
+      </p>
+      <div>
+        <a href="#"><img src="Images/twitter.svg" alt="twitter" /></a>
+        <a href="#"><img src="Images/Linkedin.svg" alt="Linkedin" /></a>
+        <a href="#"><img src="Images/medium.svg" alt="medium" /></a>
+        <a href="#"><img src="Images/github.svg" alt="github" /></a>
+        <a href="#"><img src="Images/angellist.svg" alt="angellist" /></a>
+      </div>
```

Comment:
- [ ] Please kindly put all the icons group in a `ul` tag and each icon in an `li` tag, and each should have a link(`a` tag).